test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/AnonymizedExportTests.cs

```diff
@@ -168,13 +220,31 @@ public async Task GivenInvalidConfiguration_WhenExportingAnonymizedData_ThenBadR
             Assert.Single(_metricHandler.NotificationMapping[typeof(ExportTaskMetricsNotification)]);
         }
 
-        [Fact]
+        [SkippableFact]
+        public async Task GivenAGroupIdNotExisted_WhenExportingGroupAnonymizedData_ThenBadRequestShouldBeReturned()
+        {
+            Skip.If(!_isUsingInProcTestServer, "Not using in-process fhir server.");
```

Comment:
nit: Curious to know if there is any specific reason we don't want to execute these tests against an actual fhir-server instance?

Reply:
I think because it's hard to turn on Anonymized Export. Right now we do it in fixture, with interface substitution, but you can't do that for real server.
Plus MetricHandler, there is no way to easy way to check metric handler execution on server side (it's easy if your server is in same process, you share memory, but when server is hosted somewhere in azure, it's a bit hard).

---
In reply to: [580684676](https://github.com/microsoft/fhir-server/pull/1672#discussion_r580684676) [](ancestors = 580684676)

Reply:
If we want to enable anonymizedExport only for our test servers, we should be able to update our deployment steps to provision them with the config enabled. If I remember right, we do something similar for cors settings by having a corstestconfiguration.json file that we read from and deploy to our servers. 

Agreed with the MetricHandler point. But I feel that we don't need to test for the MetricHandler notification in every test. We just need one or two tests to make sure it works as expected.

---
In reply to: [580687236](https://github.com/microsoft/fhir-server/pull/1672#discussion_r580687236) [](ancestors = 580687236,580684676)